lib/css_parser/parser.rb

```diff
@@ -7,6 +7,30 @@ class RemoteFileError < IOError; end
   # Exception class used if a request is made to load a CSS file more than once.
   class CircularReferenceError < StandardError; end
 
+  # We have a Parser class which you create and instance of but we have some
+  # functions which is nice to have outside of this instance
+  module ParserFx
```

Comment:
I was thinking I could do that but I think the Parser class does so much already, and that class is public API so if I add a class method to Parser it public and open for use but I don't want that. 

Also I am thinking we should maby change the public API to something like CssParser::Document.new, And on this instance we have method like add_block, load_url, each_rule, find_by_selector, etc.

Reply:
could make the methods private with `private_class_method` or `class << self` and `private`
ParserFx is also not private so kinda similar
if they are unrelated methods it's nice to get them out of the parser, but not sure they really are
ideally I'd want to keep the general structure the same so the upgrade instructions are just "replace method call a with b and change the args", but if you see a good reason then 👍 since it's v2 anyways

Reply:
I also want to keep the structure the same so it's just upgrade  and your good.  

I need a way to share it with RuleSet instance so I don't think  private class method is gooding to work